app/js/components/common/Header.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Link, IndexLink } from 'react-router';
+
+const NUMBER_OF_COLUMNS = 3;
+
+// Dropdown data to be displayed
+const menu = [
+  'Inpatient ward',
+  'Isolation ward',
+  'Laboratory',
```

Comment:
Did you intentionally hard code these? I thought you would make a REST call to dynamically fetch them.